### PR TITLE
Add terraform / tfhint

### PR DESCRIPTION
**Pull request by kyleburton**

### CODE REVIEW

File: inlineplz/linters/__init__.py [hunk 1]
```diff
@@ -641,6 +642,19 @@ def vendored_path(path):
...
         "run_per_file": False,
     },
+    "tflint": {
+        "install": [
+            ["brew", "install", "tflint"],
+        ],
+        "help": ["tflint", "--help"],
+        "run": ["tflint", "--format=json"],
+        "rundefault": ["tflint", "--format=json"],
+        "dotfiles": [],
```
Comment (guykisel) [inline inlineplz/linters/__init__.py]: we should support the tool's dotfiles - https://github.com/wata727/tflint#configuration

File: inlineplz/linters/__init__.py [hunk 2]
```diff
@@ -641,6 +642,19 @@ def vendored_path(path):
         "autorun": True,
         "run_per_file": False,
     },
+    "tflint": {
+        "install": [
+            ["brew", "install", "tflint"],
```
Comment (guykisel) [inline inlineplz/linters/__init__.py]: let's also `go get`

**Comment by guykisel** [comment]
@kyleburton is this PR obsolete?

**Comment by kyleburton** [comment]
Yeah, I think this PR is superseded by others that have already been merged.  Thanks!

**Pull request closed without merging**